examples/layout_with_shader/pipeline.ex

```diff
@@ -0,0 +1,140 @@
+defmodule Membrane.VideoCompositor.Examples.LayoutWithShader.Pipeline do
+  @moduledoc false
+
+  use Membrane.Pipeline
+
+  require Membrane.Logger
+
+  alias Membrane.H264
+  alias Membrane.VideoCompositor.{Context, InputState, Resolution}
+  alias Req
+
+  @impl true
+  def handle_init(_ctx, _opt) do
+    spec =
+      child(:video_compositor, %Membrane.VideoCompositor{
+        framerate: 30
+      })
+
+    spec_2 = [
+      child({:video_src, 0}, %Membrane.File.Source{
+        location: "samples/testsrc.h264"
+      })
+      |> child({:input_parser, 0}, %H264.Parser{
+        output_alignment: :nalu,
+        generate_best_effort_timestamps: %{framerate: {30, 1}}
+      })
+      |> child({:realtimer, 0}, Membrane.Realtimer)
+      |> via_in(Pad.ref(:input, 0), options: [input_id: "input_0"])
+      |> get_child(:video_compositor),
+      get_child(:video_compositor)
+      |> via_out(:output,
+        options: [resolution: %Resolution{width: 1920, height: 1080}, output_id: "output_1"]
+      )
+      |> child(:output_parser, H264.Parser)
+      |> child(:output_decoder, H264.FFmpeg.Decoder)
+      |> child(:sdl_player, Membrane.SDL.Player)
+    ]
+
+    Process.send_after(self(), :register_shader, 10)
+    Process.send_after(self(), :add_input, 5000)
+    Process.send_after(self(), :add_input, 10_000)
+    Process.send_after(self(), :add_input, 15_000)
+    Process.send_after(self(), :add_input, 20_000)
+
+    {[spec: spec, spec: spec_2], %{videos_count: 1}}
```

Comment:
`Membrane.Core` changes element mode/state to `playing` when the whole spec is processed . Without linking outputs, the spec isn't considered processes, so VC doesn't go into `playing` mode. Outputs need to receive RTP streams to link RTP.SessionBin to VideoCompositor bin output. In the case of spawning VideoCompositor in a single spec with inputs/outputs, VC won't go into a `playing` state linking outputs, that won't link before receiving RTP streams, that won't be sent by VideoCompsitor, since it's not in `playing` mode. Basically, it's a classical deadlock.
I'll try to document this better.